pyramid_oereb/lib/config.py

```diff
@@ -63,11 +63,14 @@ def get_themes():
         result = []
         plrs = Config._config.get('plrs')
         if plrs and isinstance(plrs, list):
+            position = 1
             for theme in plrs:
```

Comment:
`for position, theme in enumerate(plrs, start=1):`

Reply:
Done